## TASK: Using Group 3, how many cases did Italy and Russia each record in April?

### The following steps will be used to extract information from our CSV files:
- Import Pandas
- Create a function to read the desired data from csv file
- Extract data using the function
- Find the difference for each country

## Import Pandas

In [43]:
import pandas as pd

## Create function to extract desired data from CSV file

In [44]:
initial_date = {}
final_date = {}

def add_case(csv_file,dict):
    df = pd.read_csv(csv_file)
    dict['Russia'] = int(df['infected'][df['countries']=='Russia'])
    dict['Italy'] = int(df['infected'][df['countries']=='Italy'])      

## Extract Data

In [45]:
add_case('covid_help_data_03-Apr-2020 00-40-28.csv',initial_date)
add_case('covid_help_data_30-Apr-2020 10-02-28.csv',final_date)

# Russia Cases

In [46]:
final_date['Russia']-initial_date['Russia']

102950

# Italy cases

In [47]:
final_date['Italy']-initial_date['Italy']

88349

# Task Solution
The total cases recorded for each country in the month of April is:
- Russia - 102950
- Italy - 88349